src/main/java/com/google/step/datamanager/UserManager.java

```diff
@@ -7,12 +7,13 @@
 public interface UserManager {
 
   /**
-   * Returns a User object when the user with the email logs in.
+   * Returns a User object when the user with the given email. Creates a User object if it does not
+   * exist.
    *
-   * @param email email of the user who logs in.
+   * @param email email of the user.
    * @return User object with the email.
    */
-  public User userLogin(String email);
+  public User readUser(String email);
```

Comment:
I think usually when you're reading an object by something other than it's id, it's better to put that in the name e.g. `readUserByEmail`.